evm-template/Cargo.toml

```diff
@@ -98,9 +98,7 @@ substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk", t
 
 # Polkadot
 pallet-xcm = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false, tag = "polkadot-v1.10.0" }
-polkadot-cli = { git = "https://github.com/paritytech/polkadot-sdk", features = [
-	"rococo-native",
-], tag = "polkadot-v1.10.0" }
+polkadot-cli = { git = "https://github.com/paritytech/polkadot-sdk", tag = "polkadot-v1.10.0" }
```

Comment:
One open question I have(and this applies to all `Cargo.toml` files) is that there's no `paseo-native` flag on the polkadot sdk cli [create](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/cli/Cargo.toml), there's only `rococo-native` and `westend-native`. Not sure how what impact this has and wether we should one of those two.

Reply:
"the paseo runtime is not in the polkadot-sdk. Native runtimes are not used anymore so there shouldn't be any impact"